upload-server/testing/cli_test.go

```diff
@@ -289,7 +289,7 @@ func TestGetFileDeliveryPrefixDate(t *testing.T) {
 		"data_stream_id":    "test_stream",
 		"data_stream_route": "test_route",
 	}
-	metadata.Cache.SetConfig("v2/test_stream-test_route.json", &validation.ManifestConfig{
+	metadata.Cache.SetConfig("v2/test_stream_test_route.json", &validation.ManifestConfig{
```

Comment:
to stick with this new convention update this to match.

```suggestion
	metadata.Cache.SetConfig("v2/test-stream_test-route.json", &validation.ManifestConfig{
```

Reply:
Done